### ISSUE TITLE
Nothing loads (at least for me)

### ISSUE THREAD

**Issue by sylviaonthehighway** [ruffle-rs/ruffle#3388]
**Describe the bug**
I made a little AS2 test in Flash, and wrapped it in HTML. When I opened it in Chrome, the play button popped up, and then, nothing. There was just a white screen. I don't think it's because the HTML or .SWF file was too complicated or something, it was just a button and the text "ok...". 

**Expected behavior**
I expected the button I made to come up, really nothing else to expect.

**Is the problem with the Ruffle desktop app, extension, or self-hosted version?**
This is a problem with the extension.

**What platform are you using?**
I am using the extension on Google Chrome, and Windows 10 OS.

**Comment by jm-daemon** [comment]
I just became aware of this project when I saw a swf heavy forum use it to playback its posts, of which I found most drew up a white screen. I proceeded to install the chrome extension even though it was built on the same nightly as the web sites, but nothing changed. Also chrome still does not feel it can play swf directly as any attempt to load the raw link prompts a download. I see nothing in the extension to fix this.

**Comment by SilasJohnsonYT** [comment]
ok, i'll just try inserting that ruffle string directly into the html

**Comment by SilasJohnsonYT** [comment]
that did not work

**Comment by RArunRaja** [comment]
@SilasJohnsonYT  Hi there! Please try the code in the following link!

https://stackoverflow.com/questions/65465492/how-to-embed-ruffle-flash-player-emulator-into-html-file

I hope it will help to resolve the issue.

**Comment by SilasJohnsonYT** [comment]
It did not work. It was able to load a soccer game, but not my actual flash thing.

**Comment by RArunRaja** [comment]
So, you mean...you can see the flash file, But not all the functions. Am I right?

**Comment by SilasJohnsonYT** [comment]
No. My guess it that the code loads a specific swf file, and a swf file that isn't mine. I'll just attach the HTML file so that you can see what I mean.
[AS2 Booktopia HTML.zip](https://github.com/ruffle-rs/ruffle/files/6030346/AS2.Booktopia.HTML.zip)
My guess is that since I'm not hosting a website, nor do I have a webserver, the HTML code for ruffle is acting up.

**Comment by Lord-McSweeney** [comment]
The SWF file works fine on the desktop version of Ruffle.
